api/app/course_responses.go

```diff
@@ -66,9 +66,11 @@ func (rs *CourseResource) newCourseListResponse(courses []model.Course) []render
 
 // SheetPointsResponse is response for performance on a specific exercise sheet
 type SheetPointsResponse struct {
-	AquiredPoints int `json:"acquired_points" example:"58"`
-	MaxPoints     int `json:"max_points" example:"90"`
-	SheetID       int `json:"sheet_id" example:"2"`
+	AquiredPoints int  `json:"acquired_points" example:"58"`
+	Graded        bool `json:"graded" example:"true"`
```

Comment:
Can we have a more verbose comment about the difference between `Graded` and `SheetGraded`?

Reply:
For every successful submission, one row is inserted into table `grades`, one row into table `submissions` – With 0 points hard coded, and with tutor_id 1, again hard coded. However, the DB schema does not reflect if a submission has been graded with 0 points, or if it has not been graded at all. 

`database/sheet_store.go: PointsForUser` is used to create `TaskPointsResponse` objects for every task.
`Graded` is then a just a predicate if the `tutor_id` is _not_ equal to 1. Because then, the task hasn't been graded.

`database/course_store.go: PointsForUser` aggregates all tasks of a sheet. `SheetGraded` aggregates the points of the tasks which have been graded. (`COALESCE(SUM(t.max_points) FILTER(WHERE g.tutor_id <> 1), 0) AS "sheet_graded",`). Therefore a boolean does not suffice here.

If you don't like the separation of `Graded` and `SheetGraded`. `Graded` could be eliminated and encoded using `SheetGraded`.

Reply:
Then renaming `SheetGraded` to `AchievedPoints` would make things more clear. This would also resolve the confusion about `bool` and `int`.

About the tutor_id = 1 hard-coded, seems like a bad way of initializing it. Sorry for bad code.

Reply:
> SheetGraded aggregates the points of just the graded tasks (based on the same predicate as above)

This should be `AquiredPoints`.

> COALESCE(SUM(t.max_points) FILTER(WHERE g.tutor_id <> 1), 0) AS "sheet_graded",)

This is *not*  `AquiredPoints`. Can this  be `AchievablePoints` then.

Reply:
`AchievablePoints` sounds great to me. This should work for both, tasks, and sheets, thus replacing `Graded` and `SheetGraded` altogether.

The request would then return:
- `AquiredPoints`: How many points are earned.
- `MaxPoints`: How many points can be earned overall.
- `SheetID`
- `AchievablePoints`: How many points can be earned, considering which tasks have been graded yet.

`AchievablePoints` and `MaxPoints` will be exactly the same IFF all tasks of a sheet have been graded.